docs/upgrade.html

```diff
@@ -23,13 +23,17 @@ <h5><a id="upgrade_280_notable" href="#upgrade_280_notable">Notable changes in 2
 <ul>
     <li>
         The 2.8.0 release added a new method to the Authorizer Interface introduced in
-            <a href="https://cwiki.apache.org/confluen ce/display/KAFKA/KIP-679%3A+Producer+will+enable+the+strongest+delivery+guarantee+by+default">KIP-679</a>.
+            <a href="https://cwiki.apache.org/confluence/display/KAFKA/KIP-679%3A+Producer+will+enable+the+strongest+delivery+guarantee+by+default">KIP-679</a>.
         The motivation is to unblock our future plan to enable the strongest message delivery guarantee by default.
         Custom authorizer should consider providing a more efficient implementation that supports audit logging and any custom configs or access rules.
     </li>
-</ul>
-
-<ul>
+    <li>
+        IBP 2.8 introduces topic IDs to topics as a part of
+        <a href="https://cwiki.apache.org/confluence/display/KAFKA/KIP-516%3A+Topic+Identifiers">KIP-516</a>.
+        When using ZooKeeper, this information is stored in the TopicZNode. If the cluster is downgraded to a previous IBP or version,
+        future topics will not get topic IDs and it is not guaranteed that topics will retain their topic IDs in ZooKeeper.
+        This means that upon upgrading again, topics some or all topics will be assigned new IDs.
```

Comment:
"topics some" : Do you mean "some topics"?

Reply:
Good catch. :)